Aquality.Appium.Mobile.Template/Aquality.Appium.Mobile.Template.SpecFlow/Features/Demo.feature

```diff
@@ -0,0 +1,11 @@
+﻿Feature: Demo
+
+@demo
+Scenario: I try to login with invalid credentials
+	Given I can open 'Login Screen' view
```

Comment:
It's probably better to change the "Given" message, isn't it?
```suggestion
	Given 'Login Screen' view is opened
```

Reply:
I'll change it to "I open '()' view", because "is opened" would be better for Then steps

Reply:
maybe "I opened  '()' view"? Following the Gherkin notations: 
https://cucumber.io/docs/gherkin/reference/#given